packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -817,25 +834,23 @@ export class ListItem
   };
 
   focusCell = (focusEl: HTMLTableCellElement, saveFocusIndex = true): void => {
-    const { contentEl, actionsStartEl, actionsEndEl, parentListEl } = this;
+    const { parentListEl } = this;
 
     if (saveFocusIndex) {
       focusMap.set(parentListEl, null);
     }
 
     const focusedEl = getFirstTabbable(focusEl);
 
-    [actionsStartEl, contentEl, actionsEndEl]
-      .filter((el) => el && !el.hidden)
-      .forEach((tableCell, cellIndex) => {
-        const tabIndexAttr = "tabindex";
-        if (tableCell === focusEl) {
-          focusEl === focusedEl && tableCell.setAttribute(tabIndexAttr, "0");
-          saveFocusIndex && focusMap.set(parentListEl, cellIndex);
-        } else {
-          tableCell.removeAttribute(tabIndexAttr);
-        }
-      });
+    this.getGridCells().forEach((tableCell, cellIndex) => {
+      const tabIndexAttr = "tabindex";
+      if (tableCell === focusEl) {
+        focusEl === focusedEl && tableCell.setAttribute(tabIndexAttr, "0");
```

Comment:
If we need to update the attribute vs the prop, can you add a comment about its intentional usage?

Reply:
Updated to use property and added a comment.